addons/purchase_mrp/models/stock_rule.py

```diff
@@ -0,0 +1,15 @@
+from markupsafe import Markup
+
+from odoo import models, _
+
+
+class StockRule(models.Model):
+    _inherit = 'stock.rule'
+
+    def _notify_responsible(self, procurement):
+        origin_orders = procurement.values.get('group_id').mrp_production_ids if procurement.values.get('group_id') else False
+        if origin_orders:
+            notified_users = procurement.product_id.responsible_id.partner_id | origin_orders.user_id.partner_id
+            notification_msg = Markup(" ").join(Markup("%s") % user._get_html_link(f'@{user.display_name}') for user in notified_users)
```

Comment:
Could you print only partner name, not full computed name ? Thank you
![image](https://github.com/user-attachments/assets/f20931d0-19ea-403f-8998-1597fde0d427)